From dfc6fee7d19f9f861decaf503345ac836695ff0d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 09 Sep 2021 18:15:25 +0300
Subject: [PATCH] [SW-0] gaudi2: arc: fix the sram base

Gaudi2 ARC maps 16 regions to access the SoC.
The region 1 (2nd) is the SRAM access.
Current setting moves the SRAM start to 16MB into the SRAM and makes the
first 16MB of SRAM unused.
Additionally, it makes all the code checking the boundaries to not work
correctly.
Fix the SRAM base address.

Change-Id: I3c9d12a22d98f8d5f529de6506005625d61bab69
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index d616396..a53a541 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -34,7 +34,7 @@
  */
 #define PPBOOT_DCCM_BASE	0x2706402C
 
-#define SDRAM_BASE		0x11000000
+#define SDRAM_BASE		0x10000000
 #define HL_SPI_BASE		0x26000000
 #define HL_CFG_BASE		0xF0000000
 #define I2C_SLAVE_DEFAULT_INTR_MASK	0x48FF
